### ISSUE TITLE
Profile image is missing alt text

### ISSUE THREAD

**Issue by srimanh** [kalviumcommunity/devlinks-02#2]
The avatar <img> in src/components/Header.jsx has no alt attribute. Add descriptive alt text.

**Comment by srimanh** [comment]
Replaced: this repo has been re-themed. Fresh issues incoming.
